usaspending_api/search/v2/views/search_elasticsearch.py

```diff
@@ -65,8 +64,15 @@ def post(self, request):
                 )
             )
 
-        if validated_payload["sort"] not in validated_payload["fields"]:
-            raise InvalidParameterException("Sort value not found in fields: {}".format(validated_payload["sort"]))
+        payload_sort_key = validated_payload["sort"]
```

Comment:
Since this is being pulled out in a variable, might make sense to use the variable on line 92.

Reply:
Done